Sources/ComposableArchitecture/UIKit/AlertStateUIKit.swift

```diff
@@ -0,0 +1,80 @@
+#if canImport(UIKit) && !os(watchOS)
+import UIKit
+
+@available(iOS 13, *)
+@available(macCatalyst 13, *)
+@available(macOS, unavailable)
+@available(tvOS 13, *)
+@available(watchOS, unavailable)
+extension AlertState {
+  /// UIKit helper generating UIAlertController from corresponding AlertState
+  ///
+  ///     class ParentViewController: UIViewController {
+  ///        let store: Store<ParentState, ParentAction>
+  ///        let viewStore: ViewStore<ViewState, ViewAction>
+  ///        var cancellables: Set<AnyCancellable> = []
+  ///        private weak var alertView: UIAlertController?
+  ///        ...
+  ///        func viewDidLoad() {
+  ///          ...
+  ///          viewStore.publisher
+  ///            .settingsAlert
+  ///            .sink { [weak self] alert in
+  ///              guard let self = self else { return }
+  ///              if let alert = alert {
+  ///                let alertView = alert.toUIAlertController(send: {
+  ///                  self.viewStore.send(.settings($0))
+  ///                })
```

Comment:
This would then change to:
```suggestion
  ///                let alertView = UIAlertController(state: alert, send: {
  ///                  self.viewStore.send(.settings($0))
  ///                })
```